sage-ios/SAGE/SAGE/RootController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  RootController.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 10/6/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+
+class RootController: UIViewController {
+    
+    override func viewDidAppear(animated: Bool) {
+        super.viewDidAppear(animated)
+        self.setUpView()
+        if LoginHelper.userIsLoggedIn() {
+            if LoginHelper.userIsVerified() {
+                self.pushRootTabBarController()
+            } else {
+                self.pushUnverifiedViewController()
+            }
+        } else {
+            self.pushLoginViewController()
+        }
+    }
+    
+    func setUpView() {
```

Comment:
is this still needed? if not you can just remove this (since we arent making any network requests or anything